includes/objects/revision.php

```diff
@@ -706,19 +707,21 @@ public function update_composer_package($mode = 'add')
 		titania::_include('tools/composer_package_manager', false, 'titania_composer_package_helper');
 		$package_helper = new titania_composer_package_helper();
 
-		if (!titania::$config->composer_vendor_name || !titania_types::$types[$this->contrib->contrib_type]->create_composer_packages || !$package_helper->packages_dir_writable())
+		if (!titania_types::$types[$this->contrib->contrib_type]->create_composer_packages
+			|| !$package_helper->packages_dir_writable()
+			|| empty($this->contrib->contrib_package_name))
 		{
 			return;
 		}
-		$package_manager = new titania_composer_package_manager($this->contrib->contrib_id, $this->contrib->contrib_name_clean, $this->contrib->contrib_type, $package_helper);
+		$package_manager = new titania_composer_package_manager($this->contrib->contrib_id, $this->contrib->contrib_package_name, $this->contrib->contrib_type, $package_helper);
 
 		if ($mode == 'add')
 		{
-			$package_manager->add_release($this->revision_version, $this->attachment_id, true);
+			$package_manager->add_release(json_decode($this->revision_composer_json, true), $this->attachment_id, true);
 		}
 		else
 		{
-			$package_manager->remove_release($this->revision_version);
+			$package_manager->remove_release($this->revision_composer_json['version']);
```

Comment:
This is still a string and not an array. Last error here https://tracker.phpbb.com/browse/CUSTDB-697 refers to this.